onnx/defs/__init__.py

```diff
@@ -120,3 +121,15 @@ def get_function_ops() -> List[OpSchema]:
 
 
 SchemaError = C.SchemaError
+
+def register_schema(op: OpSchema):
+    name: str = op.name
+    domain: str = op.domain
+    ver: int = op.since_version
```

Comment:
```suggestion
    version: int = op.since_version
```

readability: Avoid abbreviations - they add cognitive load to readers.

> Do not use abbreviations that are ambiguous or unfamiliar to readers outside the project, and do not abbreviate by deleting letters within a word.

https://google.github.io/styleguide/pyguide.html#316-naming
https://google.github.io/styleguide/cppguide.html#General_Naming_Rules